kratos/processes/simple_mortar_mapper_process.h

```diff
@@ -0,0 +1,474 @@
+#include "includes/model_part.h"
+#include "geometries/point.h"
+
+/* Custom includes */
+#include "includes/mortar_classes.h"
+
+/* Custom utilities */
+#include "utilities/exact_mortar_segmentation_utility.h"
+#include "utilities/mortar_utilities.h"
+#include "utilities/math_utils.h"
+
+namespace Kratos
+{
+///@name Kratos Globals
+///@{
+
+///@}
+///@name Type Definitions
+///@{
+    
+///@}
+///@name  Enum's
+///@{
+
+#if !defined(HISTORICAL_VALUES)
+#define HISTORICAL_VALUES
+    enum HistoricalValues {Historical = 0, NonHistorical = 1};
+#endif
+    
+///@}
+///@name  Functions
+///@{
+    
+template< int TDim, int TNumNodes, class TVarType, HistoricalValues THist> 
+class SimpleMortarMapperProcess
+        : public Process
+{
+public:
+    ///@name Type Definitions
+    ///@{
+    
+    /// Pointer definition of SimpleMortarMapperProcess
+    KRATOS_CLASS_POINTER_DEFINITION(SimpleMortarMapperProcess);
+    
+    typedef Point<3>                                     PointType;
+    typedef Node<3>                                       NodeType;
+    typedef Geometry<NodeType>                        GeometryType;
+    typedef Geometry<PointType>                  GeometryPointType;
+    typedef ModelPart::NodesContainerType           NodesArrayType;
+    typedef ModelPart::ConditionsContainerType ConditionsArrayType;
+    
+    // Type definition for integration methods
+    typedef GeometryData::IntegrationMethod      IntegrationMethod;
+    
+    // Auxiliar geometries
+    typedef Line2D2<PointType>                            LineType;
+    typedef Triangle3D3<PointType>                    TriangleType;
+    
+    // Component type
+    typedef VariableComponent< VectorComponentAdaptor<array_1d<double, 3> > > component_type;
+
+    ///@}
+    ///@name Life Cycle
+    ///@{
+
+    /// Default constructor.
+    SimpleMortarMapperProcess( 
+        ModelPart& rThisModelPart,
+        TVarType& ThisVariable
+        ): mrThisModelPart(rThisModelPart),
+           mOriginVariable(ThisVariable),
+           mDestinyVariable(ThisVariable)
+    {
+    }
+    
+    SimpleMortarMapperProcess( 
+        ModelPart& rThisModelPart,
+        TVarType& OriginVariable,
+        TVarType& DestinyVariable
+        ): mrThisModelPart(rThisModelPart),
+           mOriginVariable(OriginVariable),
+           mDestinyVariable(DestinyVariable)
+    {
+    }
+
+    /// Destructor.
+    ~SimpleMortarMapperProcess() override
+    = default;
+
+    ///@}
+    ///@name Access
+    ///@{
+
+    ///@}
+    ///@name Inquiry
+    ///@{
+
+    ///@}
+    ///@name Input and output
+    ///@{
+
+    ///@}
+    ///@name Friends
+    ///@{
+
+    ///@}
+    ///@name Operators
+    ///@{
+
+    void operator()()
+    {
+        Execute();
+    }
+    
+    ///@}
+    ///@name Operations
+    ///@{
+    
+    void Execute() override
+    {
+        KRATOS_TRY;
+
+        // Defining tolerance
+        const double tolerance = std::numeric_limits<double>::epsilon();
```

Comment:
Is this a good idea?
It will be different on different machines, therefore it might influence the solution in corner cases! I used this but realized it does not always work

Reply:
What do you recommend?

Reply:
I use 1e-15
But I am not 100% sure abt this, the projection I use it not yet bulletproof

What I mean / suggest is to use sth that is consistent across machines